QueryBuilder.Test/QueryBuilder.Typed/Join.UnitTests.cs

```diff
@@ -5,6 +5,7 @@ namespace QueryBuilder.UnitTests.QueryBuilder.Typed
 {
     using System;
     using global::Azure.DigitalTwins.Core;
+    using global::QueryBuilder.Test.Generated;
```

Comment:
Why is this `global::` this is usually only needed for the Azure.* dependencies.

Reply:
That bugged me too. I've played around with different namespaces -- the `global::` is required to resolve the namespace anytime it begins with `QueryBuilder.`. My theory is that there's some compiler confusion since we've got the `QueryBuilder` type exposed in the main assembly.